web-registry/src/components/organisms/ProjectTopSection/ProjectTopSection.tsx

```diff
@@ -69,16 +75,12 @@ function ProjectTopSection({
   const apiServerUrl = process.env.REACT_APP_API_URI;
 
   const project = data?.projectByOnChainId || data?.projectByHandle; // TODO: eventually just projectByOnChainId
-  const metadata = project?.metadata; // TODO: this is from postgres metadata - needs to be from metadata resolver instead
```

Comment:
This was the key pain point. Even though we fetched the IRI metadata in the ProjectDetails component, this was re-assigning the metadata from the postgres table (similar, but not reflective of on-chain metadata updates).